Client/game_sa/CBikeHandlingEntrySA.cpp

```diff
@@ -0,0 +1,56 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        game_sa/CHandlingEntrySA.cpp
+ *  PURPOSE:     Vehicle handling data entry
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+CBikeHandlingEntrySA::CBikeHandlingEntrySA()
+{
+    // Create a new interface and zero it
+    m_pBikeHandlingSA = new tBikeHandlingDataSA;
```

Comment:
If possible use smart pointers.

Reply:
Smart pointers are cool, but i don't see a reason to use it there

Reply:
Afaik in new code its recommended(or maybe required? not sure) to use them.